use line buffered mode for stdout

We observerd duplicated syslog messages without this.
Seems to be a systemd bug. 

Index: new/lxcfs.c
===================================================================
--- new.orig/lxcfs.c
+++ new/lxcfs.c
@@ -3183,6 +3183,8 @@ int main(int argc, char *argv[])
 	newargv[cnt++] = argv[1];
 	newargv[cnt++] = NULL;
 
+	setlinebuf(stdout);
+
 	if (!cgfs_setup_controllers())
 		goto out;
 
